src/_langs/en/updates/2015-08-10-edit-html-in-the-console-panel.markdown

```diff
@@ -0,0 +1,40 @@
+---
+rss: false
+layout: update
+published: true
+title: Edit HTML in the Console Panel
+date: 2015-08-10
+article:
+  written_on: 2015-08-10
+  updated_on: 2015-08-10
+authors:
+- umarhansa
+collection: updates
+type: tip
+category: tools
+product: chrome-devtools
+featured-image: /web/updates/images/2015-08-10-edit-html-in-the-console-panel-of-devtools/gplus-8f4a2ba4.png
+source_name: DevTips
+source_url: https://umaar.com/dev-tips/60-console-edit-html/
+teaserblocks:
+- heading: ""
+  description: ""
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+- heading: ""
+  description: ""
+  image: ""
+permalink: /updates/2015/08/10/edit-html-in-the-console-panel.html
+---
+<img src="/web/updates/images/2015-08-10-edit-html-in-the-console-panel-of-devtools/console-edit-html.gif" alt="Edit HTML in the Console Panel of DevTools">
+
+The DOM node context menu you may be familiar with from the Elements panel is present in the Console Panel also.

```

Comment:
This is a lot to digest. Try this

"The DOM node context menu, which you may recognize from the elements panel, is also present  in the console panel."